tests/integration/current_catalog/test_current_catalog.py

```diff
@@ -0,0 +1,32 @@
+from tests.integration.base import DBTIntegrationTest, use_profile
+
+
+class TestCurrentCatalog(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return "current_catalog"
+
+    @property
+    def models(self):
+        return "models"
+
+    def unity_catalog_enabled(self):
+        results = self.run_sql(
+            'SET spark.databricks.unityCatalog.enabled',
```

Comment:
super nit: use double quote

Reply:
We need to apply black for test files once https://github.com/dbt-labs/dbt-spark/pull/297 is merged.
I intentionally left them to check the diff with the upstream test cases easily.